### PR TITLE
first version of updated readme

### PR DESCRIPTION
**Pull request by lucatume**
Update the tests readme file

### CODE REVIEW

**Comment by barryhughes** [comment]
We've entered final zip testing for 18.01 and ideally won't merge any other changes unless essential. Is it possible to target 18.02 instead?

File: src/Tribe/API.php [hunk 1]
```diff
@@ -61,6 +61,15 @@ public static function updateEvent( $event_id, $args ) {
 			$args['ID'] = $event_id;
 			$args['post_type'] = Tribe__Events__Main::POSTTYPE;
 
+			// allow for the change of the date and the status in the same update request
+			if (
+				isset( $args['post_date'], $args['post_status'] )
+				&& in_array( $post->post_status, array( 'draft', 'pending', 'auto-draft' ) )
```
Comment (sc0ttkclark) [inline src/Tribe/API.php]: future here too

Comment (sc0ttkclark) [reply src/Tribe/API.php]: future not needed actually

Comment (sc0ttkclark) [reply src/Tribe/API.php]: i removed it for you :)

**Comment by lucatume** [comment]
Closing this to reopen https://github.com/moderntribe/the-events-calendar/pull/1759

**Pull request closed without merging**